numpy/_core/src/multiarray/array_method.c

```diff
@@ -343,11 +343,11 @@ fill_arraymethod_from_slots(
                 }
             }
             if (i >= meth->nin && NPY_DT_is_parametric(res->dtypes[i])) {
-                PyErr_Format(PyExc_TypeError,
-                        "must provide a `resolve_descriptors` function if any "
-                        "output DType is parametric. (method: %s)",
-                        spec->name);
-                return -1;
+                // PyErr_Format(PyExc_TypeError,
+                //         "must provide a `resolve_descriptors` function if any "
+                //         "output DType is parametric. (method: %s)",
+                //         spec->name);
+                // return -1;
```

Comment:
This should not be merged!!! I noticed we cannot actually not pass `resolve_descriptors` in most cases, at least if the dtype is parametric (since it's passed as output)! Should we special case somehow?